gravity-forms/gw-multiple-entries-list-field.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Multiple Entries by List Field
+ * https://gravitywiz.com/
+ *
+ * Create multiple by entries based on the rows of a List field. All other field data will be duplicated for each entry.
+ * List field inputs are mapped to Admin-only fields on the form.
+ *
+ * Plugin Name:  Gravity Forms - Email Domain Validator
```

Comment:
Plugin name needs updating. 😄

Reply:
I forgot to update the name after copy and pasting 🤦‍♂️😅. It's done. Thanks